lib/compiler/doc/src/compile.xml

```diff
@@ -620,7 +620,21 @@ module.beam: module.erl \
             option is internal to the compiler and can be changed or
             removed at any time without prior warning.</p>
           </item>
-	</taglist>
+
+          <tag><c>line_coverage</c></tag>
+          <item>
+            <p><marker id="line_coverage"/>
+            Instrument the compiled code for line coverage.</p>
+          </item>
+
+	  <tag><c>force_line_counters</c></tag>
```

Comment:
Why is it called `force_...`? From just reading the docs the main difference from `line_coverage` seems to be that this option counts the number of times a line is executed, while `line_coverage` does not? I assume there is something else also that happens?

Reply:
I've rewritten the description to make it clearer what is happening.